plugins/aea-cli-ipfs/aea_cli_ipfs/registry.py

```diff
@@ -152,34 +151,17 @@ def register_item_to_local_registry(
 
 
 def fetch_ipfs(
-    item_type: str,
     public_id: PublicId,
     dest: str,
-    remote: bool = True,
 ) -> Optional[Path]:
     """Fetch a package from IPFS node."""
-    if remote:
-        ipfs_tool = IPFSTool(get_ipfs_node_multiaddr())
-    else:
-        ipfs_tool = IPFSTool(addr=DEFAULT_IPFS_URL_LOCAL)
-
+    ipfs_tool = IPFSTool(get_ipfs_node_multiaddr())
     try:
-        package_hash: Optional[str] = public_id.hash
+        package_hash: str = public_id.hash
     except ValueError:
-        package_hash = (
-            None if remote else get_ipfs_hash_from_public_id(item_type, public_id)
-        )
-
-    if package_hash is None:
         raise HashNotProvided(f"Please provide hash; Public id {public_id}.")
 
-    try:
-        ipfs_tool.check_ipfs_node_running()
-    except NodeError:  # pragma: nocover
-        if not remote:
-            ipfs_tool.daemon.start()
-        else:
-            raise Exception(f"Cannot connect to node with addr: {ipfs_tool.addr}")
+    ipfs_tool.check_ipfs_node_running()
```

Comment:
This can still raise an exception, right? And I assume since we don't catch it, the output will be a bit more cryptic than before. This is assuming that the user sees the message of this exception.